pkg/controller/awsfederatedaccountaccess/awsfederatedaccountaccess_controller.go

```diff
@@ -581,11 +594,24 @@ func (r *ReconcileAWSFederatedAccountAccess) cleanFederatedRoles(reqLogger logr.
 
 	roleName := currentFAA.Spec.AWSFederatedRole.Name + "-" + uidLabel
 
+	configMap, err := controllerutils.GetOperatorConfigMap(r.client)
+	if err != nil {
+		log.Error(err, "failed retrieving configmap")
+	}
+	ifFedramp, err := controllerutils.IsFedramp(configMap)
+	if err != nil {
+		log.Error(err, "Unable to verify if cluster is fedramp")
+	}
```

Comment:
Wouldn't we want to return an error if we can't read the config map and cancel reconcilation?
I see these lines duplicated multiple times in the PR, probably worth encapsulating in their own function?